src/hooks/accounts/readActiveAccount.tsx

```diff
@@ -1,17 +1,18 @@
 import { ApolloCache } from '@apollo/client';
-import { GetActiveAccountQueryResponse, GET_ACTIVE_ACCOUNT } from './queries/useGetActiveAccountQuery';
+import {
```

Comment:
Do you want to move this to `lib/`?

Reply:
I'd suggest moving all the files in the base of `hooks/accounts` into `lib/` or their respective folders too.

Reply:
Please check how balances are implemented, or consult the Readme's UI architecture section for tips on folder structure.

Reply:
done